resources/inputmaps/apple-media-keys.json

```diff
@@ -3,7 +3,9 @@
   "idmatcher": "AppleMediaKeys.*",
   "mapping":
   {
-    "KEY_PLAY": "play_pause",
+    "KEY_PLAY": "play",
+    "KEY_PAUSE": "pause",
+    "KEY_PLAY_PAUSE": "play_pause",
```

Comment:
Slightly worried about unintended side-effects. Did remotes using this config file always emit KEY_PLAY_PAUSE, and did these keys just match KEY_PLAY before?